data/mods/gennext/moves.js

```diff
@@ -296,7 +296,6 @@ let BattleMovedex = {
 		},
 		willCrit: true,
 		accuracy: true,
-		onTryHitPriority: 10,
```

Comment:
You've removed a lot of `Priority` entries. Can you tell me why? Are they all confirmed unnecessary?

Reply:
``moves.js`` doesn't support Priority!